docs/setup.md

```diff
@@ -193,12 +227,18 @@ below to bind a key to the "Back" function. We recommend ``[ctrl]+[`]``.
 
 Run the following commands in the directory `verification/l4v/`:
 
-```
+```sh
 mkdir -p ~/.isabelle/jedit/macros
 cp misc/jedit/macros/goto-error.bsh ~/.isabelle/jedit/macros/.
 ```
```

Comment:
I've never actually done a comparison, how does our macro differ to the built-in Isabelle one that has since been released? Does it make sense to continue suggesting and maintaining our version?

Reply:
I still prefer ours because it jumps to the end of the file if there is no error, so the UX for updating proofs is much nicer.